apps/studio/components/interfaces/Settings/Logs/Logs.types.ts

```diff
@@ -94,6 +94,7 @@ export type QueryType =
   | 'pg_upgrade'
   | 'pg_cron'
   | 'pgbouncer'
+  | 'unified'
```

Comment:
this seems redundant

Reply:
did we converge the unified logs into the pre existing logging query logic?

Reply:
i think it might have been part of your previous experiments? have removed nonetheless